### PR TITLE
Add the requirement classes

### PR DESCRIPTION
**Pull request by jdbs9514**
### Project requirements
- Create class `Person` with  the following:
  - Instance vars: `@id`, `@name`, and `@age`.
  - Constructor with `name`,  `age`, and `parent_permission` as parameter. `name` and `parent_permission` are optional and have default values of `"Unknown"` and `true`.
  - Getters for `@id`, `@name`, and `@age`.
  - Setters for `@name` and `@age`.
  - Private method `is_of_age?` that returns `true` if `@age` is greater or equal to 18 and `false` otherwise.
  - Public method `can_use_services?` that returns `true` if person is of age or if they have permission from parents.
- Create class `Student` with the following:
  - Inherits from `Person`.
  - Constructor extends parent's constructor by adding `@classroom` and a parameter for it.
  - Method `play_hooky` that returns `"¯\(ツ)/¯"`.
- Create class `Teacher` with the following:
  - Inherits from `Person`.
  - Constructor extends parent's constructor by adding `@specialization` and a parameter for it.
  - Override `can_use_services?` so it always returns `true`.
- Each class should be saved in a separate file.

### CODE REVIEW

File: student.rb [hunk 1]
```diff
@@ -0,0 +1,16 @@
+require_relative './person'
+
+class Student
+  attr_accesor :classroom
+
+  include person
```
Comment (zieeco) [inline student.rb]: - Please, for readability's sake, kindly make the Student class inherit from the parent class Person.
`class Student < Person`

File: teacher.rb [hunk 2]
```diff
@@ -0,0 +1,16 @@
+require_relative './person'
+
+class Teacher
+  attr_accesor :specialization
+
+  include person
```
Comment (zieeco) [inline teacher.rb]: - Please, for readability's sake, kindly make the Teacher class inherit from the parent class Person.
`class Teacher < Person`

**Review by zieeco — CHANGES_REQUESTED** [review]
Hi @jdbs9514,


Great job so far! 👏 
You've done well, however, there are some issues that you still need to work on to go to the next project, but you are almost there!


## To highlight 🎯

- No linter errors.✔️
- Descriptive PR. ✔️


## Required Changes ♻️
_Check the comments under the review._





## Optional suggestions
- N/A

**_Every comment with the **[OPTIONAL]** prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._**


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: student.rb [hunk 3]
```diff
@@ -1,10 +1,8 @@
 require_relative './person'
 
-class Student
+class Student < person
```
Comment (zieeco) [inline student.rb]: - The person is a class and should be spelled as  `Person`. please fix this before you merge

File: teacher.rb [hunk 4]
```diff
@@ -1,10 +1,8 @@
 require_relative './person'
 
-class Teacher
+class Teacher < person
```
Comment (zieeco) [inline teacher.rb]: - The person is a class and should be spelled as  `Person`. please fix this before you merge

**Review by zieeco — APPROVED** [review]
# status: APPROVED :green_circle: :100: 

Hi, @jdbs9514, 


68747470733a2f2f6d656469612e67697068792e636f6d2f6d656469612f78543078657a514755357843444a754350652f67697068792e676966


**Good job so far 👍 Your project is complete!  Please check the inline comment to fix it before you merge.**
Congratulations! 🎉 


## Highlights ⚡ 

✅ Descriptive PR 👍 
✅ Linters are passing 👏 


## Optional suggestions
- N/A

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: student.rb (hunk 3 above)
Comment (jdbs9514) [reply student.rb]: thanks i did it

File: teacher.rb (hunk 4 above)
Comment (jdbs9514) [reply teacher.rb]: thanks i did it

**Pull request merged**